node-project-template/lint-tests.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
-
-focused_tests="$(grep -r -n 'fdescribe(\|fit(' ./test)"
```

Comment:
because that was manually linting which the jasmine eslint plugin already does for you